Source/ACE/Entity/Landblock.cs

```diff
@@ -509,6 +510,25 @@ public void EnqueueBroadcast(Position pos, float distance, params GameMessage[]
         }
 
         /// <summary>
+        /// Enqueues a message for global broadcast, thread safe
+        /// </summary>
+        /// <param name="msg"></param>
+        public void EnqueueGlobalBroadcast(params GameMessage[] msgs)
+        {
+            // Atomically checks and sets the broadcastQueued bit --
+            //    guarantees that if we need a broadcast it will be enqueued in the world-managers broadcast queue exactly once
+            if (Interlocked.CompareExchange(ref broadcastQueued, 1, 0) == 0)
+            {
+                WorldManager.BroadcastQueue.EnqueueAction(new ActionEventDelegate(() => SendGlobalBroadcasts()));
```

Comment:
The only thing for certain is EnqueueBroadcast was not the right way to do a global as it does positional checks and only broadcasts to the landblock + allowed range the originator of the message was on.